packages/xo-web/src/common/intl/messages.js

```diff
@@ -1823,6 +1823,7 @@ const messages = {
   availableTemplateVarsTitle: 'Available template variables',
   templateNameInfo: 'the VM\'s name. It must not contain "_"',
   templateIndexInfo: "the VM's index, it will take 0 in case of single VM",
+  templateSshInfo: "SSH key corresponding to the key title.",
```

Comment:
Same.
And avoid punctuation
```suggestion
  templateSshInfo: "SSH key corresponding to the key title",
```